.circleci/orb.yml

```diff
@@ -0,0 +1,75 @@
+version: "2.1"
+orbs:
+  orb-tools: circleci/orb-tools@12.3
+
+filters: &filters
+  tags:
+    only: /.*/
+
+jobs:
+  test-schema-check:
+    resource_class: medium
+    docker:
+      - image: cimg/openjdk:23.0.2
+      - image: cimg/postgres:14.16
+        environment:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_HOST_AUTH_METHOD: trust
+    steps:
+      - checkout
+      - run:
+          name: "Build Skemium Binary"
+          command: |
+            mvn package assembly:single -DskipTests
+            ls ./target
+            mv ./target/skemium-*-jar-with-dependencies.jar /tmp/skemium.jar
+            echo '#!/bin/bash
+            java -jar /tmp/skemium.jar "$@"' > /tmp/skemium
+            chmod +x /tmp/skemium
+      - run:
+          name: "Create Database"
+          command: |
+            psql -U postgres -h localhost -p 5432 -c "CREATE DATABASE testdb;"
+      - run:
+          name: "Migrate Database"
+          command: |
+            psql -d testdb -U postgres -h localhost -p 5432 -f ./orb/test/migrations/0001_setup.sql
+      - skemium/check_schemas:
+          tableName: "resource"
+          pgDBName: "testdb"
+          schemaDir: orb/test/schemas
+  pack-and-publish:
```

Comment:
The `pack` part of this I have tested by replacing the relevant part in the main `config.yml`. The publish part I obviously never got around to testing...

Reply:
We have a custom implementation of this so that we can replace the version placeholder in the `check_schemas.yml` commands; that way we can publish both the binary and the new version of the orb in the same pipeline / on semver releases.